iphone/Classes/TiUIScrollableView.m

```diff
@@ -169,6 +173,12 @@ -(void)layoutSubviews
 [wrapperView setViewName: TI_STRING(@"scrollable.wrapper.view%lu", (unsigned long)[[self subviews] count])]; \
 [wrapperView addSubview:view]; \
 
+
+-(void)removeSubview:(nonnull UIView *)view
+{
+    [[view superview] removeFromSuperview];
```

Comment:
If we remove the superview of the supplied view from the superview..wouldn't that remove the scrollableview from it's parent? I'm sure it works well, just trying to understand.